File: src/main/kotlin/net/merayen/elastic/uinodes/list/midi_in_1/Info.java
package net.merayen.elastic.uinodes.list.midi_in_1;

import net.merayen.elastic.uinodes.BaseInfo;

public class Info implements BaseInfo {
	@Override
	public String getName() {
		return "MIDI input";
	}

	@Override
	public String getDescription() {
		return "MIDI from devices";
	}

	@Override
	public String[] getCategories() {
		return new String[]{"Interface"};
	}
}

File: src/main/kotlin/net/merayen/elastic/uinodes/list/from_midi_1/Info.kt
package net.merayen.elastic.uinodes.list.from_midi_1

import net.merayen.elastic.uinodes.BaseInfo

class Info : BaseInfo {
	override fun getName() = "From Midi"
	override fun getDescription() = "Convert midi to discrete signals"
	override fun getCategories() = arrayOf("core")
}